src/plugin.json

```diff
@@ -1,7 +1,7 @@
 {
   "type": "datasource",
   "name": "X-Ray",
-  "id": "grafana-x-ray-datasource",
+  "id": "grafana-x-ray-datasource-private-mirror",
```

Comment:
should this be removed?

Reply:
ohh, thanks for finding this @kevinwcyu!